crates/astria-sequencer/src/authority/component.rs

```
@@ -48,14 +48,26 @@ impl Component for AuthorityComponent {
         Ok(())
     }
 
-    #[instrument(name = "AuthorityComponent:begin_block", skip(_state))]
-    async fn begin_block<S: StateWriteExt + 'static>(
-        _state: &mut Arc<S>,
-        _begin_block: &BeginBlock,
-    ) {
+    #[instrument(name = "AuthorityComponent::begin_block", skip(state))]
+    async fn begin_block<S: StateWriteExt + 'static>(state: &mut Arc<S>, begin_block: &BeginBlock) {
```

Should probably return an error and let tower-abci take down the app gracefully